src/js/components/Search/Search.js

```diff
@@ -13,11 +14,16 @@ const Search = () => {
   const { t } = useTranslation();
   const dispatch = useDispatch();
   const searchSections = useSelector((state) => state.query.querySections);
+  const isFetchingData = useSelector((state) => state.query.isFetchingData);
 
   useEffect(() => {
     dispatch(makeGetKatsuPublic());
   }, []);
 
+  const scrollToTop = () => {
```

Comment:
move this outside of component (so below the last import line) to avoid redefining it every render